src/main/java/io/appium/java_client/pagefactory/DefaultElementByBuilder.java

```diff
@@ -166,27 +166,17 @@ protected By buildMobileNativeBy() {
         Optional<HowToUseLocators> howToUseLocatorsOptional = ofNullable(howToUseLocators);
 
         By result = null;
```

Comment:
is this variable still used?

Reply:
Nope, it's not used, have deleted it now.